### PR TITLE
Change Word Converter to not parse ANTLR

### PR DESCRIPTION
**Pull request by jskeet**
(See #494)

This means we lose bookmarks, but it's really straightforward.

Note: the tests won't pass together yet as there's global state to
fix, but I'll sort that separately.

### CODE REVIEW

**Comment by jskeet** [comment]
(Not ready to merge, but ready to discuss.)

File: standard/types.md [hunk 1]
```diff
@@ -31,8 +31,10 @@ A reference type is a class type, an interface type, an array type, a delegate t
 
 ```ANTLR
 reference_type
+    // This is a line comment 1
```
Comment (jskeet) [inline standard/types.md]: Whoops - this change was just for the purposes of testing. Obviously I'd remove it if we went down this route...

**Comment by jskeet** [comment]
This is now ready for review, one commit at a time. (We could separate it into multiple commits if we really wanted to.)

**Comment by BillWagner** [comment]
I gave this a review this morning.  I like the direction here. Simpler is certainly going to help us maintain this over time.

**Comment by jskeet** [comment]
@BillWagner: I've rebased and resolved the changes, so if you're happy to approve, I'll merge... and then start fixing the long lines :)

**Review by BillWagner — APPROVED** [review]
This LGTM @jskeet 

Great to see us moving forward.

**Pull request merged**